docs/release-notes/release-notes-0.19.0.md

```diff
@@ -75,6 +76,11 @@
 ## Performance Improvements
 
 * Log rotation can now use ZSTD 
+## Misc
```

Comment:
nit: missing new line above. 

But also can just put the entry under `Code Health` I'd say.

Reply:
Done.